.github/workflows/pull-request-test.yml

```diff
@@ -112,7 +112,7 @@ jobs:
       if: failure()
       uses: ./actions/download-test-artifacts
       env:
-        RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
+        RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES_1900 }}
```

Comment:
I guess these values will be reverted before merging, right?

Reply:
Well... I'd find it the cleanest solution to create new values files and put them into new secrets every time we have to change the values. But currently this would break some assumptions in our component repos, so for the time being I'll just use this for testing, revert back once the PR is ready, update the secret to make the PR pass (thereby breaking all other CI pipelines) and merge this PR asap.

Reply:
ah, right! The classical conundrum of the global configuration... yeah, I don't see many solutions, the only other one would be to include a timestamp/version in these variables (e.g. CI_RENKU_VALUES_20210202), but then we'll have to deal with the garbage...
I like your approach better.